semeio/jobs/rft/gendata_rft.py

```diff
@@ -157,9 +160,12 @@ def run(
 
             # Aggregate dataframe for all wells and report steps.
             dframes.append(trajectory_df)
+        else:
+            failed = True
 
-    if csvfile is not None:
+    if csvfile is not None and dframes:
```

Comment:
How about wrapping this in a `try/except` and inject some logging if it fails, so it is easier for the user to debug?

Reply:
Fixed.